src/components/form/widgets/input-field.ts

```diff
@@ -71,19 +72,19 @@ function getInputType(schema: any): InputType {
     return 'text';
 }
 
-function getStepSize(schema: any): 'any' | number {
+function getStepSize(schema: FormSchema): 'any' | number {
     if (isNumberType(schema) && schema.multipleOf) {
-        return parseFloat(schema.multipleOf) || 'any';
+        return parseFloat(schema.multipleOf as unknown as string) || 'any';
     }
 
     if (isIntegerType(schema)) {
-        return parseInt(schema.multipleOf, 10) || 1;
+        return parseInt(schema.multipleOf as unknown as string, 10) || 1;
```

Comment:
We could get rid of these ugly typecasts by doing something like `String(schema.multipleOf)`

Reply:
Or checking if it is actually a string or not before trying to convert it :slightly_smiling_face:

Reply:
Fixed in fixup